src/test/kotlin/oncall/domain/DevelopersTest.kt

```diff
@@ -0,0 +1,69 @@
+package oncall.domain
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertThrows
+
+class DevelopersTest {
+    @Test
+    fun `throws exception when duplicate developer names`() {
+        // Given
+        val namesWithDuplicates = listOf(Developer("A"), Developer("B"), Developer("A"), Developer("C"), Developer("D"))
+
+        // When & Then
+        assertThrows<IllegalArgumentException> {
+            Developers(namesWithDuplicates)
+        }
+    }
+
+    @Test
+    fun `throws exception when less than 5 developers`() {
+        // Given
+        val lessThanMinimum = listOf(Developer("A"), Developer("B"), Developer("C"), Developer("D"))
+
+        // When & Then
+        assertThrows<IllegalArgumentException> {
```

Comment:
It's same exception that gets thrown when developer names are duplicated.
I think it does not guarantee that this exception was actually caused by having less than 5 developers.
Can we compare the exception message as well? If you prefer not to assert message in tests, What would be a better way you think?